app/models/verification_attempt.rb

```diff
@@ -13,6 +13,9 @@
 #  index_verification_attempts_on_client_id  (client_id)
 #
 class VerificationAttempt < ApplicationRecord
+  VALID_FILE_TYPES = [".jpg", ".pdf", ".png"].freeze
```

Comment:
I remember Shannon saying that it's good to omit PDF. Plus if you want this to match the VALID_MIME_TYPES then it might be good to make sure each of those is represented here.